sdk/communication/Azure.Communication.Common/src/EntraTokenCredential.cs

```diff
@@ -15,8 +16,12 @@ namespace Azure.Communication
     /// </summary>
     internal sealed class EntraTokenCredential : ICommunicationTokenCredential
     {
+        private const string TeamsExtentionsScopePrefix = "https://auth.msft.communication.azure.com/";
+        private const string EntraScopePrefix = "https://communication.azure.com/clients/";
```

Comment:
Let's rename to `ComunicationClientsScopePrefix`? Both Teams and pure ACS are using Entra.